### DISCUSSION TITLE
Symfony 5.3 Could not found type with alias "Query" error

### DISCUSSION THREAD

**Discussion by ivanaareon** [overblog/GraphQLBundle#904] [Q&A]
Hello,

Since I've upgraded from Symfony 5.1 to 5.3 it apparently can't see the Query.type.yaml file, I'm getting error "Could not found type with alias "Query" error" in every GraphQL endpoint call. When I switch back to symfony 5.1 it works again.

This is working in linux docker POD in a Windows 10 OS.

I'd appreciate any help.

Thanks.

**Comment by ivanaareon** [comment]
this is not a bug from this bundle, just a docker linux POD bug.

ls -i -h

![image](https://user-images.githubusercontent.com/74661456/142000241-e030dac5-2812-463b-b74a-048aa9c61051.png)

We can close it. Thanks.

**Reply by martijn-moveyou** [reply]
@ivanaareon Your comment is incomplete and very unclear. In what way does a docker linux POD bug prevent symfony from finding a file? This is starting to look like a bug in the overblog graphql package. For some reason, the Query.types.yaml file is not found and the Queries in that file are not being registered.
